x-pack/test/functional/config.js

```diff
@@ -86,7 +86,7 @@ export default async function ({ readConfigFile }) {
         '--xpack.maps.enableDrawingFeature=true',
         '--xpack.reporting.queue.pollInterval=3000', // make it explicitly the default
         '--xpack.reporting.csv.maxSizeBytes=2850', // small-ish limit for cutting off a 1999 byte report
-        '--stats.maximumWaitTimeForAllCollectorsInS=1',
+        '--usageCollection.maximumWaitTimeForAllCollectorsInS=1',
```

Comment:
Remove the legacy config schema revealed that one. We were wrongly using the old `stats` prefix instead of `usageCollection` here. It didn't seem to have any actual impact though, maybe we should just remove this option from the functional config?